src/others/sysmon/lv_sysmon.h

```diff
@@ -60,6 +60,28 @@ void lv_sysmon_show_performance(lv_display_t * disp);
  */
 void lv_sysmon_hide_performance(lv_display_t * disp);
 
+/**
+ * Dump the FPS data recorded between the last and current dump call.
+ * @param disp      target display, NULL: use the default
+ */
+void lv_sysmon_dump_performance(lv_display_t * disp);
```

Comment:
Small nit here, since we're using the performance prefix for the other functions, this one should read 

```suggestion
void lv_sysmon_performance_dump(lv_display_t * disp);
```

Reply:
It make sense, i modify it now.